pkg/resources/schema_acceptance_test.go

```diff
@@ -170,10 +171,10 @@ func TestAcc_Schema_basic(t *testing.T) {
 				ResourceName:    "snowflake_schema.test",
 				ImportState:     true,
 				ImportStateCheck: importchecks.ComposeAggregateImportStateCheck(
-					importchecks.TestCheckResourceAttrInstanceState(helpers.EncodeSnowflakeID(id), "name", id.Name()),
-					importchecks.TestCheckResourceAttrInstanceState(helpers.EncodeSnowflakeID(id), "database", databaseId.Name()),
-					importchecks.TestCheckResourceAttrInstanceState(helpers.EncodeSnowflakeID(id), "with_managed_access", "false"),
-					importchecks.TestCheckResourceAttrInstanceState(helpers.EncodeSnowflakeID(id), "is_transient", "false"),
+					importchecks.TestCheckResourceAttrInstanceState(id.FullyQualifiedName(), "name", id.Name()),
```

Comment:
just a thought here: I wonder if we really make the right move with such a change. We could just handle the encode for 1-part identifiers the right way and then from the resource logic it would always be the same: encode - decode. Now we have some resources directly with FQN and some with encoding/decoding logic whereas it would be sufficient to hide this logic inside encode/decode. Let's chat on Monday.

Reply:
used `helpers.EncodeResourceIdentifier`